target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-sxtsq-5-ac.dts

```diff
@@ -65,17 +65,6 @@
 		watchdog@b017000 {
 			status = "okay";
 		};
-
-		ess-psgmii@98000 {
-			status = "okay";
-		};
-
-		edma@c080000 {
-			status = "okay";
-			phy-mode = "rgmii";
```

Comment:
Here, it used `rgmii`? Both devices, sxtsq5-ac and lhg5-ac, are single port.

Reply:
Again, I am skeptical that plain RGMII worked as it doesn't introduce the required delays.
But I have no way to test it